pkg/verify/certificate_identity.go

```diff
@@ -133,13 +138,23 @@ func NewCertificateIdentity(sanMatcher SubjectAlternativeNameMatcher, extensions
 // NewShortCertificateIdentity provides a more convenient way of initializing
 // a CertificiateIdentity with a SAN and the Issuer OID extension. If you need
 // to check more OID extensions, use NewCertificateIdentity instead.
-func NewShortCertificateIdentity(issuer, sanValue, sanRegex string) (CertificateIdentity, error) {
+func NewShortCertificateIdentity(issuer, issuerRegex, sanValue, sanRegex string) (CertificateIdentity, error) {
+	var r *regexp.Regexp
+	var err error
+
+	if issuerRegex != "" {
+		r, err = regexp.Compile(issuerRegex)
+		if err != nil {
+			return CertificateIdentity{}, err
+		}
+	}
+
 	sanMatcher, err := NewSANMatcher(sanValue, sanRegex)
 	if err != nil {
 		return CertificateIdentity{}, err
 	}
 
-	return NewCertificateIdentity(sanMatcher, certificate.Extensions{Issuer: issuer})
+	return NewCertificateIdentity(sanMatcher, r, certificate.Extensions{Issuer: issuer})
```

Comment:
It's inconsistent to have a SANMatcher for one property and separate issuer matching into separate fields. An alternative could be to have an IssuerMatcher struct as well, and then omit `Issuer` from the extensions?